cli/lib/errors.js

```diff
@@ -228,10 +224,12 @@ const invalidTestingType = {
  * @see https://github.com/cypress-io/cypress/issues/5808
  * @param {'close'|'event'} eventName Child close event name
  * @param {string} signal Signal that closed the child process, like "SIGBUS"
-*/
+ */
 const childProcessKilled = (eventName, signal) => {
   return {
-    description: `The Test Runner unexpectedly exited via a ${chalk.cyan(eventName)} event with signal ${chalk.cyan(signal)}`,
+    description: `The Test Runner unexpectedly exited via a ${chalk.cyan(eventName)} event with signal ${chalk.cyan(
+      signal
+    )}`,
```

Comment:
Why is this first call to chalk inline and the second one is spread multiline?

Reply:
Line got too long